Scoop/Controller/Ride Posting/PostRideSummaryViewController.swift

```diff
@@ -136,7 +136,12 @@ class PostRideSummaryViewController: PostRideViewController {
         
         driverInfoContainerView.snp.makeConstraints { make in
             make.height.equalTo(41)
+            make.leading.trailing.equalToSuperview().inset(-12)
```

Comment:
This feels weird, you are setting a negative inset for the superview, so this view will be wider than the superview? Is that what you intended to do?

Reply:
From the design, its supposed to be slightly wider than the stackview I think